API-Reference/Kokkos::Subview.md

```diff
@@ -0,0 +1,42 @@
+# `Kokkos::Subview`
+
+Header File: `Kokkos_Core.hpp`
+
+Alias template to deduce the type that is returned by a call to the subview function with given arguments.
+
+Usage:
+  ```c++
+  Kokkos::Subview<ViewType,Args> subView;
+  ```
```

Comment:
Why did you indent?
Same comment for all code blocks below

Reply:
I just saw that this pr was from before the switch to sphinx ...

PR is rebased on master and all info is migrated to the new subfolders.
No semantic changes made, but indentation is fixed.